exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/retry/RetryUtil.java

```diff
@@ -44,4 +47,29 @@ private RetryUtil() {}
   public static Set<Integer> retryableHttpResponseCodes() {
     return RETRYABLE_HTTP_STATUS_CODES;
   }
+
+  /**
+   * Reflectively access a {@link DefaultGrpcExporterBuilder} or {@link OkHttpGrpcExporterBuilder}
+   * instance in field called "delegate" of the instance, and set the {@link RetryPolicy}.
+   *
+   * @throws IllegalArgumentException if the instance does not contain a field called "delegate" of
+   *     type {@link DefaultGrpcExporterBuilder}
+   */
+  public static void setRetryPolicyOnDelegate(Object instance, RetryPolicy retryPolicy) {
```

Comment:
This replaces `DefaultGrpcExporterBuilder#getDelegateBuilder`. Whereas the `#getDelegateBuilder` returned the delegate builder which allowed you to call `setRetryPolicy(...)`, this replacement is more to the point and just sets the retry policy on the delegate if it can.